src/unix/solaris/mod.rs

```diff
@@ -966,10 +966,8 @@ extern {
                        serv: *mut ::c_char,
                        sevlen: ::socklen_t,
                        flags: ::c_int) -> ::c_int;
-    pub fn getpwuid_r(uid: ::uid_t,
-                      pwd: *mut passwd,
-                      buf: *mut ::c_char,
-                      buflen: ::size_t) -> *const passwd;
```

Comment:
Thanks for the catch! I have updated my patch according to the [man pages](http://docs.oracle.com/cd/E36784_01/html/E36874/getpwuid-r-3c.html).